instrumentation/khttp/build.gradle

```diff
@@ -2,6 +2,11 @@ apply plugin: 'groovy'
 apply from: "$rootDir/gradle/instrumentation.gradle"
 
 repositories {
+  mavenLocal()
+  mavenCentral()
+  maven {
+    url = uri("https://oss.sonatype.org/content/repositories/snapshots")
+  }
```

Comment:
Why do we need these three repos here?

Reply:
The repositories defined in `dependencyResolutionManagement` are lost otherwise. I think I'll revert to the previous version though, the next PR will make it a bit better once I move the repositories definition into a convention plugin.